rllib/algorithms/impala/impala.py

```diff
@@ -721,7 +702,11 @@ def training_step(self) -> ResultDict:
                     )
                 if not do_async_updates:
                     learner_results = [learner_results]
+
                 for results_from_n_learners in learner_results:
+                    if not results_from_n_learners[0]:
```

Comment:
Didn't we continue here before?

Reply:
No, we sent the empty results into the logger and tried to extract the model state from it, which wouldn't succeed. So it was pretty much a waste of time :)